lib/form.js

```diff
@@ -82,12 +78,37 @@ _.extend(Form.prototype, {
         });
         router.handle(req, res, callback);
     },
+    _configure: function (req, res, callback) {
+        req.form = req.form || {};
+        req.form.options = clone(this.options);
```

Comment:
ah it doesn't clone classes, functions and replicates circulars as they were originally